### PR TITLE
CreatePVC: correct setting of read/only access

### LINKED ISSUE #237: ReadWriteOnce set even when volume is "ro"
I'm guess that "ro" should mean read only: 

https://github.com/kubernetes-incubator/kompose/blob/7435f822ed17837a4d6f472f9a0f800abe1adc95/pkg/transformer/kubernetes/kubernetes.go#L178

Should we change this to be: 

``` diff
diff --git a/pkg/transformer/kubernetes/kubernetes.go b/pkg/transformer/kubernetes/kubernetes.go
index 6f47bdb..1ba7f38 100644
--- a/pkg/transformer/kubernetes/kubernetes.go
+++ b/pkg/transformer/kubernetes/kubernetes.go
@@ -175,7 +175,7 @@ func CreatePVC(name string, mode string) *api.PersistentVolumeClaim {
        }

        if mode == "ro" {
-               pvc.Spec.AccessModes = []api.PersistentVolumeAccessMode{"ReadWriteOnce"}
+               pvc.Spec.AccessModes = []api.PersistentVolumeAccessMode{"ReadOnlyMany"}
        } else {
                pvc.Spec.AccessModes = []api.PersistentVolumeAccessMode{"ReadWriteOnce"}
        }
```

### PR DESCRIPTION
**Pull request by dustymabe**
Fixes #237

### CODE REVIEW

File: pkg/transformer/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -175,7 +175,7 @@ func CreatePVC(name string, mode string) *api.PersistentVolumeClaim {
 	}
 
 	if mode == "ro" {
-		pvc.Spec.AccessModes = []api.PersistentVolumeAccessMode{"ReadWriteOnce"}
+		pvc.Spec.AccessModes = []api.PersistentVolumeAccessMode{"ReadOnlyMany"}
```
Comment (janetkuo) [inline pkg/transformer/kubernetes/kubernetes.go]: nit: use `api.ReadOnlyMany` and `api.ReadWriteMany` instead

**Comment by janetkuo** [comment]
Besides the nit this LGTM

File: pkg/transformer/kubernetes/kubernetes.go (hunk 1 above)
Comment (janetkuo) [inline pkg/transformer/kubernetes/kubernetes.go]: Update: I meant `api.ReadWriteOnce` (for the `else` case)

Comment (dustymabe) [inline pkg/transformer/kubernetes/kubernetes.go]: this should be fixed now

**Comment by dustymabe** [comment]
@janetkuo sorry about the delay. Have been away from the computer mostly since Friday.

**Pull request merged**